packages/playground/blueprints/src/lib/steps/install-asset.ts

```diff
@@ -0,0 +1,96 @@
+import type { UniversalPHP } from '@php-wasm/universal';
+import { writeFile } from './client-methods';
+import { unzip } from './import-export';
+
+export interface InstallAssetOptions {
+	/**
+	 * The zip file to install.
+	 */
+	zipFile: File;
+	/**
+	 * Target path to extract the main folder.
+	 * @example
+	 *
+	 * <code>
+	 * const targetPath = `${await playground.documentRoot}/wp-content/plugins`;
+	 * </code>
+	 */
+	targetPath: string;
+}
+
+/**
+ * Install asset: Extract folder from zip file and move it to target
+ */
+export async function installAsset(
```

Comment:
I initially had this in mind as an internal utility to aid other steps, I'm thinking whether this makes sense as its own step 🤔

Reply:
The term "asset" seems ambiguous as part of a public API. Internally it means:

- a zip package
- with a single folder of files in it
- to be extracted to a target path

Technically it's generic, not limited to plugins and themes - but I can't imagine what else would be considered "assets", that has the above characteristics.

There could be an `installMuPlugins` blueprint step, same as for plugins but with target `mu-plugins`.

Media files could be another kind of asset. Maybe an `installMedia` step that uses  `installAssets`. Unlike plugins/themes, it would not require a single container folder.

So I think `installAsset` by itself may not be useful to expose as a blueprint step, and can probably stay an internal helper function.

Reply:
Oh sorry I got confused. It is in the steps directory and kind of looks like a step declaration, but it actually is an internal helper. My bad!

Reply:
It would be nice to put each step in a separate file and move helpers somewhere distinct, maybe a subdirectory? Let's figure this out separately